README.md

```diff
@@ -46,6 +46,7 @@ Sizes of the `vlitejs` bundle compared to the competition:
 - [**Cast**](./src/plugins/cast/README.md) - Supports for Google Cast API.
 - [**AirPlay**](./src/plugins/airplay/README.md) - Supports for Apple AirPlay API.
 - [**Monetization**](./src/plugins/ima/README.md) - Supports for Google IMA SDK.
+- [**Hotkeys**](./src/plugins/hotkeys/README.md) - Supports for hotkeys to add shortcuts.
```

Comment:
Do we need to add the "keyboard" word ?

Reply:
Yes.
- Hotkeys is for Keyboard shortcuts
- Shortcuts is for Keyboard Shortcut and Desktop Shortcut

I did the commit: [8b41816](https://github.com/rtritto/vlite/commit/8b41816fcc3865e26ea67dfeef981a7667a61ba5)